utils/documentExists.js

```diff
@@ -0,0 +1,16 @@
+const { models } = require('../models/index');
+
+const documentExists = async (filterObj, modelResource) => {
+  try {
+    const [document] = await models[modelResource].where(filterObj);
+    if (document) {
+      return true;
+    } else {
+      return false;
+    }
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+module.exports = documentExists;
```

Comment:
Great utility function. Will definitely reuse it!

Reply:
This is indeed interesting.

Reply:
This is indeed interesting 🔥